doc/03-Configuration.md

```diff
@@ -0,0 +1,28 @@
+# Configuration
+
+This is how you configure your Icinga DB so it works right off the bat.
+
+## Database
+
+Select the database resource for your setup.
+
+## Redis
+
+Configure the Redis connections details of your Icinga host.  
+If you are running a master-master setup, the first 2 fields are for your configuration master.
+
+
+Option             | Description
+-------------------|------------------------
+Redis Host         | IP adress or FQDN, usually this is your Icinga Master host
+Redis Port         | The port over which you can connect to the Redis
```

Comment:
`to the Redis` ... either `to Redis` or yet better, `The port where the Redis server is listening on`.